requirements.txt

```diff
@@ -25,16 +25,16 @@ pytest-mock==3.11.1
 pyvo==1.4.2
 pyyaml==6.0.1
 redis==5.0.0
-ruff==0.0.292
-scipy==1.9.3
+ruff==0.1.6
```

Comment:
Do we need ruff at all, since we're keeping with pep8speaks?

Reply:
great question!  I'll keep updating but we should discuss